### PR TITLE
feat: add destination icons

### PR DESCRIPTION
**Pull request by SLROS**
# Alaska Airlines Pull Request

Added destination icons: flag-stroke, mountain-stroke and snowflake-stroke

**Resolves:** #256

## Summary:

Added missing destination icons for Google NLS work

## Type of change:

- [X] New capability
- [ ] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [X] My update follows the CONTRIBUTING guidelines of this project
- [X] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: src/icons/flag-stroke.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (blackfalcon) [inline src/icons/flag-stroke.svg:1]:
```suggestion
<svg>
```

File: src/icons/flag-stroke.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
....6108 12.0737 18.6154 12.0701 18.6204 12.0663C18.6458 12.0467 18.6838 12.0173 18.73 11.9653C18.8963 11.7814 18.9998 11.5043 19.0001 11.2418V11.2089L18.9969 11.1908ZM16.1598 11.2921C15.7089 11.4562 15.2586 11.571 14.892 11.6154H14.8909L14.8857 11.6162L14.884 11.6165L14.881 11.6168L14.8795 11.617L14.8779 11.6172C14.733 11.6393 14.5874 11.6498 14.4323 11.6498C13.9429 11.6498 13.4436 11.5396 12.9324 11.4167C12.1243 11.2146 11.2613 11.0175 10.3734 11.0175C10.1946 11.0175 10.0179 11.0257 9.84799 11.0418C9.04502 11.1105 8.17906 11.3191 7.27312 11.6623C7.16528 11.1888 7.00458 10.3791 6.84836 9.59151C6.68318 8.75886 6.51236 7.89793 6.36705 7.23533C6.31944 6.99973 6.27815 6.79997 6.24488 6.63944L6.24435 6.63709C6.23607 6.59683 6.22831 6.55943 6.22108 6.52464C6.4349 6.44372 6.61677 6.37726 6.82955 6.30937C7.67618 6.04094 8.39091 5.90855 9.01424 5.9049C9.64415 5.91976 10.2622 6.05919 10.9166 6.20682C11.6095 6.36319 12.3255 6.52464 13.0783 6.53441C13.1263 6.53558 13.1751 6.53624 13.2232 6.53624C14.24 6.53624 15.207 6.2738 15.946 5.79753C16.2281 7.25982 16.6083 9.08358 16.8662 10.3212C16.9127 10.5443 16.954 10.7428 16.9902 10.9169C16.8049 11.0292 16.491 11.1716 16.1598 11.2921Z" fill="#626B79"/>
```
Comment (blackfalcon) [inline src/icons/flag-stroke.svg:2]: Remove all `fill` color references from SVG. Please see https://github.com/AlaskaAirlines/Icons/blob/master/README.md#do

Please address with all included icons.

File: src/icons/flag-stroke.svg (hunk 1 above)
Comment (blackfalcon) [reply src/icons/flag-stroke.svg:1]: Please address with all included icons.

**Review by blackfalcon — CHANGES_REQUESTED** [review]
Please update code commit to follow our Contributing Guidelines. 

https://github.com/AlaskaAirlines/WC-Generator/blob/master/.github/CONTRIBUTING.md#conventional-commits

File: src/icons/flag-stroke.svg (hunk 2 above)
Comment (SLROS) [reply src/icons/flag-stroke.svg]: Fixed

File: src/icons/flag-stroke.svg (hunk 1 above)
Comment (SLROS) [reply src/icons/flag-stroke.svg]: @blackfalcon fixed

**Review by blackfalcon — APPROVED** [review]

**Pull request merged**